src/KnpTimeBundle.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Knp\Bundle\TimeBundle;
+
+use Knp\Bundle\TimeBundle\DependencyInjection\KnpTimeBundleExtension;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+final class KnpTimeBundle extends Bundle
+{
+    public function getContainerExtension(): ?ExtensionInterface
```

Comment:
Why is this needed? If it were called `KnpTimeExtension`, wouldn't this method be not needed?

Reply:
Learn something new everyday